class/Defaults/MergeTag/Post/PostThumbnailUrl.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Post thumbnail url merge tag
+ *
+ * Requirements:
+ * - Trigger property of the post type slug with WP_Post object
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\MergeTag\Post;
+
+use BracketSpace\Notification\Defaults\MergeTag\UrlTag;
+
+
+/**
+ * Post thumbnail url merge tag class
+ */
+class PostThumbnailUrl extends UrlTag {
```

Comment:
```suggestion
class ThumbnailUrl extends UrlTag {
```

`Post` here is redundant if it's in the `Post` namespace